src/vs/workbench/contrib/terminalContrib/suggest/test/browser/terminalSuggestAddon.integrationTest.ts

```diff
@@ -63,13 +62,14 @@ interface IRecordedSessionResizeEvent {
 	rows: number;
 }
 
-suite('Terminal Contrib Suggest Recordings', () => {
+suite('Terminal Contrib Suggest Recordings, PWSH', () => {
```

Comment:
I think these are still generic, it just depends on what provider is used within the test. I think we want to reuse this infrastructure for other shells too.